tables/grid_search.tex

```diff
@@ -1,13 +1,13 @@
 \begin{table}[h]
 	\centering
-	\caption{Tested hyperparameter values. $K_1$ is the $K$ values used for the first hyperparameter test, while $K_2$ is the $K$ values used for the second hyperparameter test.}
+	\caption{Tested hyperparameter values. $K_1$ is the $K$ values used for the random gridsearch, where we tested each $K$ value with a random chosen $\alpha$ and $\eta$, while $K_2$ is the $K$ values used for the grid search in conjunction with the bolded values in $\alpha$ and $\eta$ test.}
```

Comment:
Noget af det her med random grid search, og at der køres to grid searches burde nok også beskrives i teksten tidligere.